torch/_torch_docs.py

```diff
@@ -9922,6 +9922,50 @@ def merge_dicts(*dicts):
              [4, 6]]])
 """.format(**common_args))
 
+add_docstr(torch.split,
+           r"""
+split(input, split_size, dim=0) -> List[Tensor]
+
+Splits the tensor into chunks. Each chunk is a view of the original tensor.
+
+If :attr:`split_size_or_sections` is an integer type, then :attr:`tensor` will
+be split into equally sized chunks (if possible). Last chunk will be smaller if
+the tensor size along the given dimension :attr:`dim` is not divisible by
+:attr:`split_size`.
+
+If :attr:`split_size_or_sections` is a list, then :attr:`tensor` will be split
+into ``len(split_size_or_sections)`` chunks with sizes in :attr:`dim` according
+to :attr:`split_size_or_sections`.
+
+Args:
+    tensor (Tensor): tensor to split.
+    split_size_or_sections (int) or (list(int)): size of a single chunk or
+        list of sizes for each chunk
+    dim (int): dimension along which to split the tensor.
+
+Example::
+
+    >>> a = torch.arange(10).reshape(5,2)
+    >>> a
+    tensor([[0, 1],
+            [2, 3],
+            [4, 5],
+            [6, 7],
+            [8, 9]])
+    >>> torch.split(a, 2)
+    (tensor([[0, 1],
+                [2, 3]]),
```

Comment:
indentation looks bad? IS it good in the rendered doc?

Reply:
Ah, I think my editor re-formatted it. Fixed now.